src/test/ui/symbol-names/issue-75326.rs

```diff
@@ -0,0 +1,58 @@
+// build-fail
+// ignore-tidy-linelength
+// revisions: legacy v0
+//[legacy]compile-flags: -Z symbol-mangling-version=legacy
+//[v0]compile-flags: -Z symbol-mangling-version=v0
+//[legacy]normalize-stderr-test: "he370ae05ed270ccc" -> "SYMBOL_HASH"
+//[legacy]normalize-stderr-test: "h9756931825e237b5" -> "SYMBOL_HASH"
```

Comment:
When I kept what was passing for me locally as `normalize-stderr-64bit` and put CI's result in `normalize-stderr-32bit` then that didn't work - both local and CI are 64-bit - I made both `-test` because that worked.

Reply:
I've changed it back and checked that tests pass locally with 64bit and 32bit, so we'll see if it fails.